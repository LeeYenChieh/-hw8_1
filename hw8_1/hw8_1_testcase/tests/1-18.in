Q9aYb5rjHeCGyi9SakQfHd45ohPVg7ejRWXBhCvK vckuTNuOQjcxbgzUp2N
93SXSxlkAe4s7adrIMn19adDkcu7Z nqVzoT9tX8IQn8
Q9aYb5rjHeCGyi9SakQfHd45ohPVg7ejRWXBhCvK display
aL H2xMciHsqaQdyE6Yq
nU7K24i9Rizsa6IhnSGwWkVciprxXnVhXUMCbZJpIzM e9RTTIttmLXPOhgVr1jg
hQBnKqWA8LmGLxs9bPd3UCdNtBzn4KZbvAI5OY9D9YyPO3o JJtAd
LFslXg00arxt1FJ1NwYBcOvfh0GlSNxepU97uqHKoo EhpxkN6P3tsZ7x
2a4UKlvEaUll VKed8a2zFH78du
xxw4iCvZ5c2kc3eu VoGPRvj
WbmAXe ICtiYrulVGFWJMuyfistCx0yReB
nU7K24i9Rizsa6IhnSGwWkVciprxXnVhXUMCbZJpIzM 8TXjCMhyTdu9cup3YsRpzNXFoBWY23R2hMSyg
xxw4iCvZ5c2kc3eu lUtrV6XsCeUxugGuJaw0ALZuHlnMZIMgI4mxj5Vyx0XPOvX5hvZXah
nU7K24i9Rizsa6IhnSGwWkVciprxXnVhXUMCbZJpIzM display
aL add thickBlackBorder thickBlackBorder
xxw4iCvZ5c2kc3eu 7S2bMbQ4IDjBG6dmLQVB
