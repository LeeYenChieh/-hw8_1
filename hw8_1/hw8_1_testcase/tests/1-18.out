vckuTNuOQjcxbgzUp2N 
8TXjCMhyTdu9cup3YsRpzNXFoBWY23R2hMSyg 
